### DISCUSSION TITLE
Notifiche di Trascrizioni e Annotazioni automatiche per Separazioni/Divorzi.

### DISCUSSION THREAD

**Discussion by gcuci** [italia/ansc#915] [Miglioramenti] [answered]
Buongiorno,

segnaliamo una ridondanza nelle Notifiche che vengono prodotte, successivamente alla validazione/firma degli eventi generati per un iter di Separazione/Divorzio.

Per una verifica delle Notifiche, abbiamo realizzato un iter di separazione/divorzio, validando e firmando in sequenza gli eventi di: 'Accordo di separazione personale dei coniugi davanti all’Ufficiale di Stato Civile', 'Conferma di accordo di separazione', 'Accordo di scioglimento (o di cessazione degli effetti civili) del matrimonio (divorzio) davanti all’Ufficiale di Stato civile' e 'Conferma di accordo di divorzio'.
Per ognuno di questi eventi, vengono generate 2 Notifiche di Trascrizione (una per ognuno degli sposi).
In considerazione del fatto che solo le Conferme di Accordo di divorzio, hanno una conseguenza anagrafica, riteniamo che siano corrette solo le notifiche a fronte di questi eventi. Non dovrebbero invece essere prodotte le Notifiche per gli eventi di Accordo e di conferma dell’accordo di separazione.

In attesa di riscontro,

cordiali saluti.

**Comment by mttfranci** [comment, marked as answer]
Salve, la segnalazione è in lavorazione in relazione anche alla issue #677 e la discussione #916. Sarà presumibilmente inclusa nella prossima minor version.

Saluti

Assistenza ANSC

**Comment by gcuci** [comment]
Buongiorno,
 
segnaliamo che adesso, riprovando un iter come quello indicato, vengono correttamente generate le Notifiche di trascrizione, solo sulla Conferma dell'accordo di Divorzio (e le relative Notifiche di Annotazione automatica), ma non vengono invece più generate le Notifiche di Annotazione automatica, sulla Conferma dell'accordo di Separazione, che in precedenza venivano correttamente generate.
 
In attesa di riscontro,
 
cordiali saluti.
